integrationtests/src/test/java/org/axonframework/integrationtests/cache/EhCacheIntegrationTest.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright (c) 2010-2023. Axon Framework
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.axonframework.integrationtests.cache;
-
-import net.sf.ehcache.CacheManager;
-import net.sf.ehcache.Ehcache;
-import net.sf.ehcache.config.CacheConfiguration;
-import net.sf.ehcache.config.SizeOfPolicyConfiguration;
-import net.sf.ehcache.store.MemoryStoreEvictionPolicy;
-import org.axonframework.common.caching.Cache;
-import org.axonframework.common.caching.EhCacheAdapter;
-import org.junit.jupiter.api.*;
-
-/**
- * {@link Ehcache} specific implementation of the {@link CachingIntegrationTestSuite}.
- *
- * @author Steven van Beelen
- */
-class EhCacheIntegrationTest extends CachingIntegrationTestSuite {
```

Comment:
Why remove the old test case while we still have the EhCache 2 solution? :thinking:

Reply:
Partly accidental, but it's probably nice to keep them around.